oshi-core/src/main/java/oshi/driver/linux/proc/TaskStat.java

```diff
@@ -0,0 +1,259 @@
+/**
+ * MIT License
+ *
+ * Copyright (c) 2010 - 2020 The OSHI Project Contributors: https://github.com/oshi/oshi/graphs/contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package oshi.driver.linux.proc;
+
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import oshi.software.common.AbstractOSThread;
+import oshi.software.os.OSProcess;
+import oshi.software.os.linux.LinuxOSProcess;
+import oshi.software.os.linux.LinuxOperatingSystem;
+import oshi.util.ExecutingCommand;
+import oshi.util.FileUtil;
+import oshi.util.ParseUtil;
+import oshi.util.platform.linux.ProcPath;
+
+public class TaskStat extends AbstractOSThread {
```

Comment:
Why are we extending `AbstractOSThread`?  I think you renamed LinuxOSThread rather than the file closer to `ProcessStat`.  Which is what I suggested, but it's because I was confused!  Sorry.

This should be LinuxOSThread (like you had it) but you had it in the `driver` package hierarchy instead of the `oshi.software.os.linux` package.  You can either add to the `ProcessStat` driver package or create a new `TaskStat` package if there's not a lot of overlap.

Reply:
Sure i can do this.

Reply:
Not sure if you saw my note, I already tried to put things in the right place. Sorry for going back and forth on suggestions here, but this is normal in the process of developing a new API.

Reply:
I completely understand this and i believe great things evolve this way only. To tell you, this has been quite a bit a learning for me in terms of collaboration :). Thanks a lot for being such a supportive mentor.